editor/project_manager.cpp

```diff
@@ -1667,58 +1664,70 @@ int ProjectList::get_project_count() const {
 	return _projects.size();
 }
 
-void ProjectList::select_project(int p_index) {
+void ProjectList::_clear_project_selection() {
 	Vector<Item> previous_selected_items = get_selected_projects();
```

Comment:
Not really related to this PR, but this line makes a pointless copy. Instead it could iterate on `_selected_project_paths` and then clear it afterwads. (also the loop can be replaced with range iterator)